### ISSUE TITLE
Meta kategorie

### ISSUE THREAD

**Issue by pionl** [cesko-digital/movapp-data#114]
Aktuálně metakategorie se vložili do kategpriích ale pocitově tam nemají co dělat.

Co jsem pochopil, tak to má spojit kategorie a vytvořit "nad kategorii".

Správně má být meta kategorie ve vlastním AirTable seznamu a místo v jsonu `categories` by to mělo být třeba `meta_categories`.

Takto se to ohnulo a muselo se přidat filtrace isMetaOnly apod. Další prostor udělat chybu jak v AirTable tak i v kódu.

cc @met @kubito @vchlum

**Comment by met** [comment]
Přidávám @JuroUhlar který metakategorie řeší.
Juraj to chtěl tenhle víkend doladit, přejmenovat "metacategories" na "subcategories", čímž by to dávalo smysl.

viz vlákno ve Slacku https://cesko-digital.slack.com/archives/C03BYED3ARK/p1684659317880859

**Comment by JuroUhlar** [comment]
@pionl Yes it's a mess right now. I proposed renaming the "meta categories" to "subcategories", deleting the "meta_only" boolean, and relying on checking if the category has `phrases.length > 0` or ` subcategories.length > 0` it shouldn't have both. 

We could separate meta categories into their own table but @met is counting with multiple nesting levels in the future, so maybe having it all in one table is more flexible. Let me know what you think, open to other ideeas how to structure it. If we agree, I can fix it this weekend.

**Comment by met** [comment]
@pionl For now subcategories are not in dictionary. But the idea is in the end to show them in dictionary (still need to figure how we want to do it).
